File: bingo-api/src/Bingo.Api.Web/Teams/TeamResponse.cs
using Bingo.Api.Web.Events;
using Bingo.Api.Web.Players;
using JetBrains.Annotations;

namespace Bingo.Api.Web.Teams;

[PublicAPI]
public class TeamResponse : TeamShortResponse
{
    public EventShortResponse Event { get; set; }
    public List<PlayerResponse> Players { get; set; } = [];
}

File: bingo-frontend/src/clients/responses/teamResponse.ts
import type { PlayerResponse } from '@/clients/responses/playerResponse.ts';
import type { EventResponse } from '@/clients/responses/eventResponse.ts';

export type TeamResponse = {
  id: number;
  avatar: string;
  name: string;
  event: EventResponse;
  players?: PlayerResponse[];
}
